src/bootstrap-table.js

```diff
@@ -830,7 +830,9 @@
                     that.options.singleSelect = true;
                 }
 
+                html.push('<p>');
```

Comment:
@fenichelar yes, you're right.. we were very very busy but we are trying to close issues and merge some **PRs**.. let me test it and I will accept this **PR**